content/docs/guidelines/best-practices/tutorial-outline/index.md

```diff
@@ -8,50 +8,140 @@ weight = 5
+- **Audience**: one of our [target audiences]({{< ref "/docs/guidelines/best-practices/tutorial-audiences/index.md" >}}), the intended group of readers for your tutorial.
+- **Teaching points**: a bullet list of the key teaching points and new techniques covered in the tutorial.
+
+This information forces you to be clear on what you're writing about. It also gives teammates the necessary context to review your work.
+
+### Writing the tutorial header
+
+Next, you want to turn your key information into:
+- a concrete list of things the user will learn in the tutorial
+- a list or a mention of prerequisites
+- an introduction to your tutorial
+
+You can learn more about the composition of a header in our [tutorial structure guide]({{< ref "/docs/guidelines/best-practices/tutorial-structure/index.md" >}}).
+
+### Defining sections with descriptive headings
+
+Headings structure a document. They split it into subsections, each with a _specific teaching goal_.
+
+**Every subheading should correspond to solving a given problem, implementing a new mechanic, or answering a broad question.**
+
+You want to make sure your headings are as descriptive and appealing as possible: 
+
+- Avoid using new class names and technical titles.
+- Avoid one or two-word headings.
+- Try to capture the problem you will be solving or the game mechanic you will be adding in the section.
+
+For example, do not write "Player script." Instead, write "Coding the player's movement."
+
+Another example: do not write "Area2D node." Instead, write "Detecting hits using the Area2D node."
+
+### Adding notes and code snippets
+
+Once you've written the heading and you're satisfied with their order, add a few lines and code snippets to start shaping the content of each section.
+
+You want to focus on key teaching points and the order in which you'll teach.
+
+Write down questions the readers might have and try to do some troubleshooting. You should work with a peer at this stage to ensure you do not forget anything.
+
+---
+## Example Outline
+####Let's have a look at the outline of a tutorial that teaches how to draw strokes around 2D sprites.
+
+- **Title**: Drawing outlines around 2D sprites.
+- **Pitch** (or short description): Learn to create an outline shader for 2D sprites, with three variations: Inner, Outer, and both way strokes.
+- **Audience**: intermediate-level, developers who might have professional experience, but no knowledge of shader programming.
+- **Techniques** (teaching points):
+  - Sampling textures at an offset. `texture(TEXTURE, UV + vec2(x_offset, y_offset))`
+  - Computing and combining masks.
+  - Using `smoothstep` to soften value transitions.
+
+<!-- Shouldn't the Intro be below what you will learn? What about prerequisites?-->
+Intro: Start with examples of gameplay situations where this shader can be useful, like outlining collectibles or interactive objects on hover.
```

Comment:
You can think of this as the pitch: "this tutorial is about achieving this result, which is useful in that context." It's the short pitch for what this page is about, so people know right away if that matches what they were looking for.

Reply:
Makes sense. Ok for the edits. Thanks